.gitignore

```diff
@@ -47,7 +47,7 @@ opendap/src/test/data/test.*.*
 tds/src/test/content/thredds/logs
 tds/src/test/content/thredds/cache
 tds/src/test/content/thredds/state
-tds/src/test/content/thredds/notebooks/
+tds/src/test/content/thredds/notebooks/default_viewer.ipynb
```

Comment:
I think I'd still lean toward ignoring the whole directory with the exception of the test file, but this does seem to be fine :)